src/Util/File.cc

```diff
@@ -141,6 +217,17 @@ CARTA::FileType GuessRegionType(const std::string& path_string, bool check_conte
     return CARTA::UNKNOWN;
 }
 
+/**
+ * @details This function attempts to classify the type of a catalog table file (e.g., FITS table or VOTable)
+ * by first checking its magic number (if `check_content` is `true`) or, if content checking is
```

Comment:
Here I would also suggest "either by..." instead of "by first..." (I would say "first by...", but later in the sentence we have "or" rather than "then").